src/main/java/strategy/BalanceOperationHandlerImpl.java

```diff
@@ -0,0 +1,17 @@
+package strategy;
+
+import dao.StorageDao;
+import model.FruitTransaction;
+
+public class BalanceOperationHandlerImpl implements OperationHandler {
```

Comment:
```suggestion
public class BalanceOperationHandler implements OperationHandler {
```

same for other handlers